lib/sequelize.js

```diff
@@ -667,22 +632,16 @@ module.exports = (function() {
    *
    * @see {Model#build} for more information about callee.
    */
-  Sequelize.prototype.query = function(sql, callee, options, replacements) {
+  Sequelize.prototype.query = function(sql, callee, options) {
```

Comment:
Will also make it easier to distinguish between `options.instance` and `options.model` as callee.